### DISCUSSION TITLE
Submitted. 🎉🏆

### DISCUSSION THREAD

**Discussion by deborahtrez** [curiouslearning/CuriousReaderInteractiveFunctionsH5P#228] [#OSEQ Hackathon]
If you opened this thread, it means that you've probably finished creating and submitting your content for consideration for the winning prizes!
Congratulations on finishing your task(s)!🎉😍 Yaaayy!!
Please feel free to share your experiences with us! We would love to hear back from you!
What was most exciting? 😁✨
What gave you the most pain? 😮‍💨
What do you wish the Curious Learning or #OSEQ team did differently? 😏
Would you participate in another Curious Learning Hackathon again? 😎
Any takeaways, or pieces of advice for future participants? 🤷🏽‍♀️

**Comment by EdithTess** [comment]
@deborahtrez , I have submitted one book so far, translated and audio by our group...kindly take a look!

**Reply by EdithTess** [reply]
Edith-Stuart folder

**Reply by deborahtrez** [reply]
Hello @EdithTess ! I have just taken a look at your team's folder. Beautiful! Great work! 👏🏽👏🏽😊 I really love that you followed the translation guidelines and the script was neat and clear.

I did notice in your audio submissions, that you recorded each audio twice in the same sound byte. I wanted to ask why you chose to do it this way as opposed to once. 

Also the naming of the audio files .... say for example somebody else took over building the book using your script and audio, how would they know which audio is for which sentence? Lets assume the book builder does not understand Luganda (because it's a diverse team). 
The way to solve this and to guide the book builder to the right audio is to name the translated sentences in English. For example, if you translated "Blue hands" to Luganda as "_Emikono gya bululu_". You can name the audio file "Blue hands.m4a" so that when the book builder is uploading the audio into the interactive book they can tell that the translation is for that sentence.

**Reply by EdithTess** [reply]
@deborahtrez , Thanks for the feedback!
Kindly check it out. I added another folder

**Reply by EdithTess** [reply]
I have uploaded another book....you can take a look!
Thank you

**Reply by EdithTess** [reply]
Hide and seek!

**Comment by EdithTess** [comment]
Great!
Thank you for the reply, lets redo & rename the audio!

**Comment by nchuti1** [comment]
@deborahtrez i have submitted my lot. thank you

**Reply by deborahtrez** [reply]
That's great Nchuti.

**Comment by nchuti1** [comment]
Greetings.
I would like to know what time is the event tommorrow so that I plan my day.
Thank you.

**Reply by deborahtrez** [reply]
Hi Nchuti.

I must have made an error about the date of the award ceremony. The award ceremony will be on the 19th of August. The hackathon ended on the 12th.
All those who actively participated will receive an invitation from #OSEQ to the award ceremony. 

Kindly look out for any such communication in the next week till the 18th. 

Thank you.

![image](https://user-images.githubusercontent.com/28389950/184510565-24c794ac-07e3-4013-9e1c-7408684062c0.jpeg)

**Comment by nchuti1** [comment]
Thank for the clarification. Let me work on my schedule for that day.
Thank you,.
